.github/ISSUE_TEMPLATE/task-module-service.md

```diff
@@ -0,0 +1,26 @@
+---
+name: "Task: NixOS service module"
+about: Serivce module task for an NGI project
```

Comment:
```suggestion
```

If it's not a mandatory field let's not write anything extra here.

Reply:
Actually, I think it's mandatory. I'm testing another template and GitHub is complaining that it should not be empty:

![image](https://github.com/user-attachments/assets/cbba40b1-c481-4073-b606-7c15449dac92)